css/style.sass

```diff
@@ -0,0 +1,33 @@
+$imgWidth: 400px
```

Comment:
tak na prawdę ten plik nie powinien być w katalogu css tylko `styles` ponieważ nie jest to plik css :)